examples/custom_route_rules/terraform.tfvars.example

```diff
@@ -47,6 +47,10 @@ defined_tags = {
   "Operations.CostCenter" = "42"
 }
 
+local_peering_gateways = {
+  to_shared_svc = {}
```

Comment:
Can you please also give a full blown configuration of what goes inside the ```to_shared_svc``` ?

Reply:
in this situation for the simplicity of the example, we could imagine leaving it as is. This mean the module will create an LPG and will stay in acceptor state, until a peering request is done from another VCN for example.

Some documentation already exists that explain the various usage: do you want me to put a comment with the following link: https://github.com/oracle-terraform-modules/terraform-oci-vcn/blob/main/examples/hub-spoke/README.md#how-to-declare-one-or-many-lpg-on-the-vcn-module ?

Reply:
Please give both in the terraform.tfvars.example and comment them. Something like:

```
local_peering_gateways = {
   # leave empty in acceptor state
   # to_shared_svc = {}
   
   # include parameter details below
   #to_lpg_{
      ...
      ...
   # }
}

Maybe we should also add these in the docs?